runner/shared/src/main/scala/hedgehog/runner/Properties.scala

```diff
@@ -12,7 +12,8 @@ abstract class Properties {
   /** Allows the implementing test to be run separately without SBT */
   def main(args: Array[String]): Unit = {
     val config = PropertyConfig.default
-    val seed = Seed.fromTime()
+    val seed = Seed.fromEnvOrTime()
+    println(seed.source.renderLog)
```

Comment:
I'm wondering if it's intentional.

Reply:
The `println`? Yes, that's for showing the seed. I don't think we have a proper logger our anything here, do we? As far as I can see, that's only available in the SBT runner.

Reply:
Yeah, you're right. I don't think we have any logger in the runner project. I was just wondering if it was used for debugging and you forgot to remove. Yeah, it makes sense to show the user what seed source is used.

Reply:
In the runner there are the sbt loggers. I'm using that to log the same information. :) See here: https://github.com/hedgehogqa/scala-hedgehog/pull/157/files?file-filters%5B%5D=.scala#diff-fa948c4a13a416c93d4225a07e8eb10fL95-R96